test/visual-diff/visual-tests

```diff
@@ -607,6 +607,8 @@
       "interactive_tests": "examples/visual-tests/amp-story/amp-story-page-attachment-desktop.js"
     },
     {
+      "flaky": true,
+      // See https://percy.io/ampproject/amphtml/builds/1434487/view/95137509/375?browser=firefox&mode=diff
       "url": "examples/visual-tests/amp-date-picker/amp-date-picker.amp.html",
       "name": "amp-date-picker",
       "loading_complete_css": [
```

Comment:
Is there any other css element that can be added to this list, to indicate that the date picker is fully populated?

Reply:
Good question, I'll let you know what I find out 🤔